v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,125 @@
+
+### Delayed execution of sequentially-executed statements 
+
+The SQL statements to be executed in parallel should have a `RETURNING NOTHING` clause, while sequentially-executed statements do not have the `RETURNING NOTHING` clause. Consider a scenario where a sequentially-executed SQL statement follows a batch of parallely-executed statements within a single transaction. The sequentially-executed statement will be executed after all the parallel statements are done executing. Thus it may seem as if the sequentially-executed statement is taking longer to execute, but it's in fact waiting on the parallel statements. Even then, the total time required for parallel execution of statements followed by the sequential execution of the SQL statement should be less than time required for the sequential execution of all statements. 
+
+### Error message mismatch
+
+It is possible to see a mismatch in the error message being displayed and the statement being executed. Consider a scenario where a dependent statement is executed after a batch of independent statements, and one of the independent statements results in an error. Now because the `RETURNING NOTHING` clause is used with the independent statements, CockroachDB cannot return the error for the independent statement. The error message for the independent statement is then reported on the next dependent statement. Thus you might see a mismatch with the error message to the statement being executed. 
+
+On encountering the error, CockroachDB will stop the execution of all parallel statements and [abort and retry the transaction](transactions.html#error-handling). So even an error message mismatch occurs, the result will be the same as aborting the transaction.
+
+## Independent SQL statements
+
+As mentioned earlier, CockroachDB considers SQL statements within a single transaction to be independent if they operate on independent tables and their execution can be safely reordered without affecting their results. 
+
+For example, the following statements are considered independent since they operate on two independent tables and reordering the statements will not affect the results:
+
+~~~ sql
+> INSERT INTO a VALUES (100);
+> INSERT INTO b VALUES (100);
+~~~
+
+The following statements are not considered independent since they operate on the same table:
+
+~~~ sql
+> INSERT INTO a VALUES (100);
+> INSERT INTO a VALUES (200);
+~~~
+
+The following pairs of statements are dependent since reordering them will affect their results:
+
+~~~ sql
+> UPDATE a SET b = 2 WHERE y = 1;
+> UPDATE a SET b = 3 WHERE y = 1;
+~~~
+
+~~~ sql
+> UPDATE a SET y = true  WHERE y = false;
+> UPDATE a SET y = false WHERE y = true;
+~~~
+
+## Required Privileges
```

Comment:
Why? What does a user learn from this at the moment? These are all privileges of the statements themselves and are not specific to RETURNING NOTHING. In that way, this is very similar to views, which are orthogonal to the specific statements they use.